test/TestProjects/OperationGroupMappings/Configuration.json

```diff
@@ -0,0 +1,5 @@
+{"OutputFolder":".","Namespace":"OperationGroupMappings","LibraryName":null,"SharedSourceFolders":["..\\..\\..\\src\\assets\\Generator.Shared","..\\..\\..\\src\\assets\\Azure.Core.Shared"],"AzureArm":true,"PublicClients":true,"ModelNamespace":true,"HeadAsBoolean":false,"SkipCSProjPackageReference":true,
```

Comment:
for other examples this is inside the generated folder, any reason its here for this one?

Reply:
![image](https://user-images.githubusercontent.com/53361486/108134766-a52e0400-706b-11eb-84f5-c2bf3e816f87.png)

Modeled it after inheritance since the standalone needs the codemodel and configuration to be there

![image](https://user-images.githubusercontent.com/53361486/108134833-c131a580-706b-11eb-8571-b68b8ff9df9a.png)